core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1697,4 +1706,54 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 
 		return $label;
 	}
+
+	/**
+	 * @param array<int|string> $ids
+	 */
+	protected function preloadCurrentRecords(array $ids, string $table): void
```

Comment:
```suggestion
	protected function loadCurrentRecords(array $ids, string $table): void
```

Reply:
I don't like this suggestion. It explicitly preloads them. `loading` would imply, that if I called `$dc->getCurrentRecord()` I'd get the new one which I don't.